subset/bacnet/bacnetTests/src/main/java/FauxDeviceEngine/Analog.java

```diff
@@ -0,0 +1,245 @@
+package FauxDeviceEngine;
+
+import com.serotonin.bacnet4j.LocalDevice;
+import com.serotonin.bacnet4j.exception.BACnetServiceException;
+import com.serotonin.bacnet4j.obj.BACnetObject;
+import com.serotonin.bacnet4j.type.Encodable;
+import com.serotonin.bacnet4j.type.constructed.*;
+import com.serotonin.bacnet4j.type.enumerated.*;
+import com.serotonin.bacnet4j.type.primitive.CharacterString;
+import com.serotonin.bacnet4j.type.primitive.Real;
+import com.serotonin.bacnet4j.type.primitive.UnsignedInteger;
+
+import java.util.Map;
+
+public class Analog {
+
+    private static float presentValue = 0.1f;
+    private static String objectName = "device_run_command";
+    private static String deviceType = "";
+    private static float deadband = 14;
```

Comment:
these variables dont need to be assigned values

Reply:
should I just leave them with a null value?
It doesn't matter if they have a value or not.